home.html

```diff
@@ -0,0 +1,154 @@
+        <p>Join our virtual meeting. </p>
+    </div>
+</div>
+</section>
+
+<section id="program">
+    <div class="programContent">
+        <div class="programTitle">
+            <h2 class="programTitle1">Main Program</h2>
+            <div id="ruler"></div>
+        </div>
+            <div class="programCard">
+                <div class="card1">
+                    <img class="cardImage" src="./assets/program/icons/users-solid.svg">
+                <div class="cardtext">
+                        <h6 class="cardtitle">
+                            Friend finder
+                        </h6>
+                        <p class="cardDescription">Find players you know and people playing near you.</p>
+                </div>
+                </div>
+                <div class="card1">
+                    <img class="cardImage" src="./assets/program/icons/dice-d20-solid.svg">
+                <div class="cardtext">
+                        <h6 class="cardtitle">
+                            Dice Roler
+                        </h6>
+                        <p class="cardDescription">We have virtual Dice at your disposal.</p>
+                </div>
+                </div>
+                <div class="card1">
+                    <img class="cardImage" src="./assets/program/icons/hat-wizard-solid.svg">
+                <div class="cardtext">
+                        <h6 class="cardtitle">
+                            Dungeon masters
+                        </h6>
+                        <p class="cardDescription">Have a look at our registered Dungeon masters, see their rating and book with them a campaign.</p>
+                </div>
+                </div>
+                <div class="card1">
+                    <img class="cardImage" src="./assets/program/icons/comments-solid.svg">
+                <div class="cardtext">
+                        <h6 class="cardtitle">
+                            Forum
+                        </h6>
+                        <p class="cardDescription">Share your thoughts and opinions.</p>
+                </div>
+                </div>
+                <div class="card1">
+                    <img class="cardImage" src="./assets/program/icons/dragon-solid.svg">
+                <div class="cardtext">
+                        <h6 class="cardtitle">
+                            Cosplayers
+                        </h6>
+                        <p class="cardDescription">See how other people show up to their campaign and post your own image.</p>
+                </div>
+                </div>
+            </div>
+        <a href="#" id="seeprogram">#See Whole Program</a>
+        <button class="JoinNPConnect" id="JoinNPConnect">Join NPConnect</button>
+            
+    </div>
+</section>
+
+<section id="dungeonMaster" class="dungeonMaster">
+    <div class="speaker-heading">
+        <h1>Famous Dungeon Masters</h1>
+        <div class="ruler"></div>
+    </div>
+    <div class="speaker-contents">
+    </div>
```

Comment:
- [ ] Color of the text and font size for the speaker section should also be adjusted accordingly to match our design. Also, add the missing underline to the speaker section heading and the divider for each speaker below each speaker's `job title`.

Your design
![image](https://github.com/George7h/Module-1-capstone-project/assets/81932478/df4213e8-641f-487d-8752-b495d036273a)


Given design
![image](https://github.com/George7h/Module-1-capstone-project/assets/81932478/548cac87-9bba-47b9-b872-5c1cd68aed09)